### ISSUE TITLE
QML icon is not loaded for *.qml files

### ISSUE THREAD

**Issue by MaxRivas** [vscode-icons/vscode-icons#2800]
- [ ] I'm sure this issue is not a _duplicate_.
- Language Extensions:
- More info:

The extension advertises an icon for *.qml files however it does not seem to work now. It did with previous versions, I don't recall what is the latest version on which is worked.

I have submitted 3 images:
1- QML files that do not show the icon
![2021-07-02 13_25_10-dune (Workspace)  SSH_ z6g4devdune052 boi rd hpicorp net  - Visual Studio Code](https://user-images.githubusercontent.com/60399715/124320633-748dc900-db39-11eb-98a6-b308b7fdc18a.jpg)

2- qmldir files that do have the correct icon
![2021-07-02 13_25_43-qmldir - dune (Workspace)  SSH_ z6g4devdune052 boi rd hpicorp net  - Visual Stud](https://user-images.githubusercontent.com/60399715/124320645-7b1c4080-db39-11eb-9789-217f25eed1e3.jpg)

3- What the *.qml icon should be from the list of support files
![2021-07-02 13_26_12-ListOfFiles · vscode-icons_vscode-icons Wiki and 2 more pages - Work - Microsoft](https://user-images.githubusercontent.com/60399715/124320664-81122180-db39-11eb-971f-1b1bc7dfc008.jpg)

**Comment by sasial-dev** [comment]
As far as I'm aware, you have to set the language in VSCode for it to work.
It's in the bottom right of the UI, and then you can configure the file association.

**Comment by remcohaszing** [comment]
Indeed, the `qml` icon is applied to the `qml` language ID. You have to install an extension that adds `qml` support for the icon to show up.
